test/Feature/HLSLLib/smoothstep.16.test

```diff
@@ -0,0 +1,236 @@
+    # 0.0019378662, 0.9892578, 0.0019378662, 0.9892578, 0.0, 0.0, 1.0, 1.0
+  - Name: Result3D
+    Format: Float16
+    Stride: 8
+    ZeroInitSize: 16
+  - Name: ExpectedResult3D
+    Format: Float16
+    Stride: 8
+    Data: [ 0x17F0, 0x3BEA, 0x3C00, 0x0, 0x17F0, 0x3BEA, 0x3C00, 0x0]
+    # 0.0019378662, 0.9892578, 1.0, 0.0, 0.0019378662, 0.9892578, 1.0, 0.0
+  - Name: Result4D
+    Format: Float16
+    Stride: 8
+    ZeroInitSize: 16
+  - Name: ExpectedResult4D
+    Format: Float16
+    Stride: 8
+    Data: [ 0x17F0, 0x3BEA, 0x3AC0, 0x3BE6, 0x17F0, 0x3BEA, 0x3AC0, 0x3BE6 ]
+    # 0.0019378662, 0.9892578, 0.84375, 0.9873047, 0.0019378662, 0.9892578, 0.84375, 0.9873047
+Results:  
+  - Result: CheckResult2D
+    Rule: BufferFloatULP
+    ULPT: 2
+    Actual: Result2D
+    Expected: ExpectedResult2D
+  - Result: CheckResult3D
+    Rule: BufferFloatULP
+    ULPT: 2
+    Actual: Result3D
+    Expected: ExpectedResult3D
+  - Result: CheckResult4D
+    Rule: BufferFloatULP
+    ULPT: 2
+    Actual: Result4D
+    Expected: ExpectedResult4D
+
+DescriptorSets:
+  - Resources:
+    - Name: Min2D
+      Kind: StructuredBuffer
+      DirectXBinding:
+        Register: 0
+        Space: 0
+      VulkanBinding:
+        Binding: 0
+    - Name: Min3D
+      Kind: StructuredBuffer
+      DirectXBinding:
+        Register: 1
+        Space: 0
+      VulkanBinding:
+        Binding: 1
+    - Name: Min4D
+      Kind: StructuredBuffer
+      DirectXBinding:
+        Register: 2
+        Space: 0
+      VulkanBinding:
+        Binding: 2
+    - Name: Max2D
+      Kind: StructuredBuffer
+      DirectXBinding:
+        Register: 3
+        Space: 0
+      VulkanBinding:
+        Binding: 3
+    - Name: Max3D
+      Kind: StructuredBuffer
+      DirectXBinding:
+        Register: 4
+        Space: 0
+      VulkanBinding:
+        Binding: 4
+    - Name: Max4D
+      Kind: StructuredBuffer
+      DirectXBinding:
+        Register: 5
+        Space: 0
+      VulkanBinding:
+        Binding: 5
+    - Name: X2D
+      Kind: StructuredBuffer
+      DirectXBinding:
+        Register: 6
+        Space: 0
+      VulkanBinding:
+        Binding: 6
+    - Name: X3D
+      Kind: StructuredBuffer
+      DirectXBinding:
+        Register: 7
+        Space: 0
+      VulkanBinding:
+        Binding: 7
+    - Name: X4D
+      Kind: StructuredBuffer
+      DirectXBinding:
+        Register: 8
+        Space: 0
+      VulkanBinding:
+        Binding: 8
+    - Name: Result2D
+      Kind: RWStructuredBuffer
+      DirectXBinding:
+        Register: 9
+        Space: 0
+      VulkanBinding:
+        Binding: 9
+    - Name: Result3D
+      Kind: RWStructuredBuffer
+      DirectXBinding:
+        Register: 10
+        Space: 0
+      VulkanBinding:
+        Binding: 10
+    - Name: Result4D
+      Kind: RWStructuredBuffer
+      DirectXBinding:
+        Register: 11
+        Space: 0
+      VulkanBinding:
+        Binding: 11     
+...
+#--- end
+# UNSUPPORTED: Clang-Vulkan

```

Comment:
Same feedback as on the other PR, I think we want to use XFAIL here and make sure we have an issue tracking it.